cli/bpmetadata/tfconfig.go

```diff
@@ -281,6 +293,10 @@ func parseBlueprintRoles(rolesFile *hcl.File) ([]BlueprintRoles, error) {
 		for k, _ := range iamAttrs {
 			var iamRoles []string
 			attrValue, _ := iamAttrs[k].Expr.Value(nil)
+			if !attrValue.Type().IsTupleType() {
```

Comment:
It would be good to add some test coverage here to validate and prevent regressions.

Reply:
Added